docs/_Interface_IColonyNetwork.md

```diff
@@ -0,0 +1,858 @@
+|_submissonIndex|uint256| Index of reputation hash submission (between 0 and 11)
+
+
+
+**Return Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|minerWeight|uint256| The weight of miner reward
+
+
+
+### `createColony`
+
+Creates a new colony in the network
+
+
+
+**Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|_tokenAddress|address| Address of an ERC20 token to serve as the colony token
+
+
+
+**Return Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|_undefined_|address|
+
+
+
+### `createMetaColony`
+
+Create the Meta Colony, same as a normal colony plus the root skill
+
+
+
+**Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|_tokenAddress|address| Address of the CLNY token
+
+
+
+
+
+### `deprecateSkill`
+
+Mark a global skill as deprecated which stops new tasks and payments from using it.
+
+
+
+**Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|_skillId|uint256| Id of the skill
+
+
+
+
+
+### `getChildSkillId`
+
+Get the id of the child skill at index `_childSkillIndex` for skill with Id `_skillId`
+
+
+
+**Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|_skillId|uint256| Id of the skill
+|_childSkillIndex|uint256| Index of the `skill.children` array to get
+
+
+
+**Return Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|skillId|uint256| Skill Id of the requested child skill
+
+
+
+### `getColony`
+
+Get the number of colonies in the network
+
+
+
+**Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|_id|uint256|
+
+
+
+**Return Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|colonyAddress|address|
+
+
+
+### `getColonyCount`
+
+Get the number of colonies in the network
+
+
+
+
+
+**Return Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|count|uint256| The colony 
+
+
+
+### `getColonyVersionResolver`
+
+Get the `Resolver` address for Colony contract version `_version`
+
+
+
+**Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|_version|uint256| The Colony contract version
+
+
+
+**Return Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|resolverAddress|address| Address of the `Resolver` contract
+
+
+
+### `getCurrentColonyVersion`
+
+Returns the latest Colony contract version. This is the version used to create all new colonies
+
+
+
+
+
+**Return Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|version|uint256| The current / latest Colony contract 
+
+
+
+### `getENSRegistrar`
+
+Returns the address of the ENSRegistrar for the Network
+
+
+
+
+
+**Return Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|_undefined_|address|
+
+
+
+### `getFeeInverse`
+
+Return 1 / the fee to pay to the network. e.g. if the fee is 1% (or 0.01), return 100
+
+
+
+
+
+**Return Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|_feeInverse|uint256|
+
+
+
+### `getMetaColony`
+
+Get the Meta Colony address
+
+
+
+
+
+**Return Parameters**
+
+
+|Name|Type|Description|
+|---|---|---|
+|_undefined_|address|
```

Comment:
Badly parsed return parameter

Reply:
This one neither matches the name (no name) or type (`address`)

```
  /// @return colonyAddress The Meta colony address, if no colony was found, returns 0x0
  function getMetaColony() public view returns (address payable);
```

Reply:
I'll also fix this.